### ISSUE TITLE
Valid attribute selectors flagged as invalid properties

### ISSUE THREAD

**Issue by justsee** [rossPatton/stylint#54]
It seems [attribute presence and value selectors](http://www.w3.org/TR/css3-selectors/#attribute-representation) are being flagged as invalid properties. For instance:

```
a[href]
  outline: none;
```
when run through `stylint` generates:

```
Warning:  property is not valid
File: test.styl
Line: 1: a[href]
```

**Comment by awayken** [comment]
I see this, too, when `valid: true`.

I don't see any issue if I do this:
```stylus
a[href] {
  outline: none;
}
```

**Comment by rossPatton** [comment]
this should be fixed on the develop branch, please verify

**Comment by awayken** [comment]
This worked for me, but I ran into another weird bug. If my `a[href]` declaration came at the top of my Stylus file, then I get the following error:

```bash
D:\Sites\stylint\src\checks\duplicates.js:69
                        this.prevLine.indexOf(',') === -1 ) {
                                     ^
TypeError: Cannot read property 'indexOf' of undefined
    at Object.duplicateSelectors [as duplicates] (D:\Sites\stylint\src\checks\duplicates.js:69:17)
    at Object.test (D:\Sites\stylint\src\test.js:118:14)
    at D:\Sites\stylint\src\parse.js:50:15
    at Array.forEach (native)
    at D:\Sites\stylint\src\parse.js:46:9
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:379:3)
```

Let me know if you'd like me to create a separate bug report for this.

**Comment by rossPatton** [comment]
hah, that makes sense, and the fix should be easy actually. i just need something like `lineNum > 0` before i do that check

**Comment by rossPatton** [comment]
should be fixed in 0.9.3

**Comment by awayken** [comment]
:+1: No longer throws an error when I pull develop.

**Comment by rossPatton** [comment]
awesome
